components/collector/src/source_collectors/jenkins/job_runs_within_time_period.py

```diff
@@ -0,0 +1,44 @@
+"""Jenkins job runs within time period collector."""
+
+from datetime import datetime
+from typing import cast
+
+from collector_utilities.functions import days_ago
+from collector_utilities.type import Job
+from model import Entities, Entity, SourceMeasurement, SourceResponses
+
+from .base import JenkinsJobs
+
+
+class JenkinsJobRunsWithinTimePeriod(JenkinsJobs):
+    """Collector class to measure the amount of Jenkins jobs run within a specified time period."""
+
+    def _include_build(self, build) -> bool:
+        """Return whether to include this build or not."""
+        build_datetime = datetime.utcfromtimestamp(int(build["timestamp"] / 1000.0))
+        return days_ago(build_datetime) <= int(cast(str, self._parameter(parameter_key="lookback_days")))
```

Comment:
See earlier comment.

```suggestion
        return days_ago(build_datetime) <= int(cast(str, self._parameter("lookback_days")))
```